### ISSUE TITLE
iOS Stories performance issues and crash

### ISSUE THREAD

**Issue by SandroMachado** [Netvent/storyly-mobile#80]
## Environment Details
**Storyly SDK Version:** 1.9.2
**Device Model:** iPhone 11 Pro 256
**Device OS Version:** 14.4.2
**Additional Info:** Using WiFi connection with 400mb download and 100mb for upload.

## Description

As you can see on the videos bellow, the performance on some stories is not the best and the app crashes without any reason - this on the first video. On the second video, it works as expected.

**First Video**

https://user-images.githubusercontent.com/631739/113132927-00304a80-9217-11eb-8b20-3a5393e8838e.mov

**Second Video**

https://user-images.githubusercontent.com/631739/113132978-15a57480-9217-11eb-9cfe-11ab1df48207.mov

**Comment by levent-oral** [comment]
Hello @SandroMachado,

We've realized that the issue with related with GIF usage on stories. The performance and crash related with memory usage is observed with GIF usage.

We're working on better GIF experience. We can temporarily suggest using videos instead of GIF. Moreover, we'll let you know  when we've improved this. 

Thanks,

**Comment by levent-oral** [comment]
We've tested GIF playback using SDWebImage and FLAnimatedImage libraries. SDWebImage is our dependency to handle image operations, FLAnimatedImage is highly popular and used library for GIF playbacks.

We've tested with several GIFs from Giphy. Both libraries runs smoothly without any memory issue and render performance reduction. 

We've also tested with the GIF, uploaded in your account. SDWebImage encounters a memory usage issue that leads to crash. FLAnimatedImage encounters an observable render performance issue. So, using another library may not solve the problem here.  

To sum up, we suggest using videos instead of GIF for this type of usage. We've also tested by converting video to the GIF, realized that reduced size, bandwidth friendly, and increased performance. 

Thanks,
